web/docs/internal/gestures/gesture-modeling.md

```diff
@@ -0,0 +1,53 @@
+# GestureRecognizer - gesture modeling
+GestureRecognizer is designed to support customizable sets of gesture-components and gesture-component sequences, rather than forcing a non-customizable and limited number of gestures.  In order to simplify the requirements for gesture customization, there are three layers used when modeling gesture components.  Other aspects of GestureRecognizer have been designed to allow manage the parallel evaluation of multiple gesture components and ensure that only one per input triggers completion.
+
+For definitions to a few commonly-used terms below, see the [Terms](./terms.md) document.
+
+The three layers comprising gesture models are as follows:
+1. [`PathModel`](#pathmodel) is responsible for modeling the path of the touchpoint handled by the `ContactModel` owning it.
+2. [`ContactModel`](#contactmodel) owns a `PathModel` and is additionally responsible for validating initial state, changes in associated item, and the portion of previously-existing path to consider if continuing an active gesture sequence.
+3. [`GestureModel`](#gesturemodel) owns at least one `ContactModel` and defines the gesture-component's properties and relationships to other gesture-components.  Also defines special properties for gesture-components with an initial state that lacks active touchpoints.
+
+Topping these off is the [`GestureModelDefs`](#gesturemodeldefs) type.  This object should be defined once, with a full set of custom gesture-component definitions, as part of configuring a `GestureRecognizer` instance.
```

Comment:
This should come before the Path/Gesture/Contact Model defs above. It's also non-specific in terms of its relationship, ironically. Does it contain one or more GestureModels or something else?

Start with an overview of the concepts, then talk about the way it is implemented. A gesture model  ... has components ... has paths ... has contacts, etc etc etc. 

The concepts are not isolated from the implementation -- the design is informed by the implementation and vice-versa. But the design should be described before the details of how it is implemented.